modules/users/client/config/users.authentication.client.routes.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('users')
+    .config(routeConfig);
+
+  routeConfig.$inject = ['$stateProvider'];
+
+  function routeConfig($stateProvider) {
+    $stateProvider
+      .state('authentication', {
+        abstract: true,
+        url: '/authentication',
+        views: {
+          'main@': {
+            templateUrl: 'modules/core/client/views/layouts/fixed.one.column.client.view.html',
+            controller: 'AuthenticationController'
+          }
+        }
+      })
+      .state('authentication.signup', {
+        url: '/signup',
+        views: {
+          'row-1': {
```

Comment:
Making layouts.  Look at modules/core/client/views/layouts/fixed.one.column.client.view.html

These are reusable layouts that submodules can target.  So you can target the ui-view from the layout file.  So since ththe signup route has 2 views technically, one for the login form, and one for social auths they are injected into a layout file.